deps.bzl

```diff
@@ -3613,3 +3613,27 @@ def prysm_deps():
         sum = "h1:PeVNzgTRtWGm6fVic5i21t+n5ptPGCZuMcSPVMyTWjs=",
         version = "v0.0.0-20180321022601-755488143dae",
     )
+    go_repository(
+        name = "com_github_cloudflare_cloudflare_go",
+        importpath = "github.com/cloudflare/cloudflare-go",
+        sum = "h1:J82+/8rub3qSy0HxEnoYD8cs+HDlHWYrqYXe2Vqxluk=",
+        version = "v0.10.2-0.20190916151808-a80f83b9add9",
+    )
```

Comment:
- Update this PR, after https://github.com/prysmaticlabs/prysm/pull/7301 is merged (we need to re-run `bazel run //:gazelle -- update-repos -from_file=go.mod -to_macro=deps.bzl%prysm_deps` and make sure that clougflares roughttime is not included into project - it is removed in https://github.com/prysmaticlabs/prysm/pull/7299)

Reply:
@farazdagi I think that dep is used by geth which is why it has been added. It seems separate from roughtime. I resynced with master and no changes for me locally from running `gazelle`

Reply:
![image](https://user-images.githubusercontent.com/188194/93879422-58c0e100-fce4-11ea-87f6-771fc3b597b8.png)

I suppose go-ethereum brings that dependency, Nishant, can you please double check that every updated import is intended?